netbout-web/src/test/java/com/netbout/rest/account/TkSaveEmailTest.java

```diff
@@ -47,6 +47,10 @@
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
 public final class TkSaveEmailTest {
+    /**
+     * Email parameter name.
+     */
+    private static final transient String EMAIL = "email";
```

Comment:
@dmzaytsev please see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#redundant-constants